index.js

```diff
@@ -66,6 +66,34 @@ if (exports.FormData) {
     }
     this.parts.push({field: fieldName, file: file});
   };
+
+  /**
+   * Set new value of an existing key (first found) or adds key/value if does not already exist.
```

Comment:
I'm reading [the specification](https://developer.mozilla.org/en-US/docs/Web/API/FormData/set), and it says that `set` should override all existing values, not the first found.

>The difference between `set()` and [`append()`](https://developer.mozilla.org/en-US/docs/Web/API/FormData/append) is that if the specified key does already exist, `set()` will overwrite all existing values with the new one, whereas append() will append the new value onto the end of the existing set of values.

It seems like this behavior that developers could expect.

Reply:
No worries, I also see that case as a rare one, but on the other side, it's allowed, so I believe it is better to implement it closer to the specification.

Reply:
I have updated the method to now replace all existing values in the form data.